inc/Admin.php

```diff
@@ -226,77 +229,51 @@ private function should_show_welcome_notice(): bool {
 	}
 
 	/**
-	 * Get the Otter Blocks plugin status.
+	 * Get the WP Full Pay plugin status.
 	 *
 	 * @return string
 	 */
-	private function get_otter_status(): string {
+	private function get_wpfp_status(): string {
 		$status = 'not-installed';
 
-		if ( is_plugin_active( 'otter-blocks/otter-blocks.php' ) ) {
+		if ( is_plugin_active( 'wp-full-stripe-free/wp-full-stripe.php' ) ) {
 			return 'active';
 		}
 
-		if ( file_exists( ABSPATH . 'wp-content/plugins/otter-blocks/otter-blocks.php' ) ) {
+		if ( file_exists( ABSPATH . 'wp-content/plugins/wp-full-stripe-free/wp-full-stripe.php' ) ) {
 			return 'installed';
 		}
 
 		return $status;
 	}
 
 	/**
-	 * Run after Otter Blocks activation.
+	 * Run after WP Full Pay activation.
 	 *
 	 * @param string $plugin Plugin name.
 	 *
 	 * @return void
 	 */
-	public function after_otter_activation( $plugin ) {
-		if ( 'otter-blocks/otter-blocks.php' !== $plugin ) {
-			return;
-		}
-
-		if ( ! class_exists( '\ThemeIsle\GutenbergBlocks\Plugins\FSE_Onboarding' ) ) {
-			return;
-		}
-
-		$status = get_option( \ThemeIsle\GutenbergBlocks\Plugins\FSE_Onboarding::OPTION_KEY, array() );
-		$slug   = get_stylesheet();
-
-		if ( ! empty( $status[ $slug ] ) ) {
-			return;
-		}
-
-		// Dismiss after two days from activation.
-		$activated_time = get_option( 'nonprofit_fse_install' );
-
-		if ( ! empty( $activated_time ) && time() - intval( $activated_time ) > ( 2 * DAY_IN_SECONDS ) ) {
-			update_option( Constants::CACHE_KEYS['dismissed-welcome-notice'], 'yes' );
+	public function after_wpfs_activation( $plugin ) {
```

Comment:
Just a small suggestion, I think you can use this hook for the activation: https://developer.wordpress.org/reference/functions/register_activation_hook/